src/main/java/com/griddynamics/cloud/learning/dao/repository/RoleRepository.java

```diff
@@ -0,0 +1,11 @@
+package com.griddynamics.cloud.learning.dao.repository;
+
+import com.griddynamics.cloud.learning.dao.domain.Role;
+import org.springframework.data.repository.CrudRepository;
+
+import java.util.List;
+
+public interface RoleRepository extends CrudRepository<Role, Long> {
+
+    List<Role> findAllBy();
```

Comment:
Probably it makes sense explicitly override it to return custom collection instead of Iterable that will be returned by the default method thus making code cleaner when I call this method in my service class (and we can create streams directly)